packages/privacy-scan-runner/src/runner/runner.spec.ts

```diff
@@ -244,10 +246,14 @@ function setupProcessScanResult(): void {
         if ((privacyScanResults.error as BrowserError)?.errorType === 'BannerXPathNotDetected') {
             runState = pageScanResult.run?.retryCount >= maxFailedScanRetryCount ? 'completed' : 'failed';
         }
+        const scanError = privacyScanResults.error as ScanError;
+        if (isEmpty(scanError.errorType)) {
+            scanError.errorType = 'InternalError';
```

Comment:
Is it still possible for privacyScanResults.error to be a string? What happens here in that case?

Reply:
It is not; Maxim's previous changes modified the signature for PrivacyScanResult.error to be Error | BrowserError.